src/components/ContextBreadcrumbs.tsx

```diff
@@ -110,6 +111,7 @@ const BreadCrumb = React.memo(
       verticalAlign: 1,
       userSelect: 'none',
     }
+    const superscriptCss = { position: 'relative', left: '-2px', top: '-3px' }
```

Comment:
Actually this will create a new object reference every call to render. You have to move it to the module scope at the top of the file :).